### ISSUE TITLE
[Week 2] Deadline for project phase 1

### ISSUE THREAD

**Issue by Jefferson111** [nusCS2113-AY1920S1/forum#6]
What's the deadline for the [tasks over here](https://nuscs2113-ay1920s1.github.io/website/schedule/week2/project.html)?

Sorry, I really couldn't find the deadline anywhere else and might hav missed it.

**Comment by okkhoy** [comment]
In general, deadlines are the midnight before the tutorial of the week. Since there are no tutorials this week, we will consider anything done before next week tutorial as "on time." E.g., For Wednesday tutorial, complete the tasks by Tuesday 2359 hrs.

**Comment by Jefferson111** [comment]
Then are we allowed to one shot phase 1 of the project? E.g. Finish everything in one go like this [person](https://github.com/Parcly-Taxel/duke/tree/master). 

Or do we have to ensure that the commits need to be spread out over the month?

**Comment by okkhoy** [comment]
@Jefferson111 No you are not allowed to do it in one shot. You need to ensure incremental updates.
Please follow the instructions on the website (partially, also because things in the schedule can change).

BTW, the reference you provided is someone from CS2103 (see screenshot). They are expected to do things differently. 
![image](https://user-images.githubusercontent.com/1023494/63318996-07108f00-c34b-11e9-9429-7bbb1a6446c3.png)

**Comment by rssujay** [comment]
I would like get clarification on your reply, as to whether I would be correct in saying that generally, week X's project tasks are due at midnight before week X's tutorial lesson.

That is, is it expected of us to have completed week 2 + 3 tasks before our first tutorial in week 3?

**Comment by okkhoy** [comment]
Yes. You need to complete week2 and week3 work the midnight before the week3 tutorial.
